pkg/trace/api/endpoints.go

```diff
@@ -113,8 +113,8 @@ var endpoints = []endpoint{
 		Handler: func(r *HTTPReceiver) http.Handler { return r.debuggerProxyHandler() },
 	},
 	{
-		Pattern:   "/v0.6/config",
-		Handler:   func(r *HTTPReceiver) http.Handler { return http.HandlerFunc(r.handleConfig) },
+		Pattern:   "/v0.6/get_config",
```

Comment:
Why `get_`? Lets drop that please, unless there is a strong conventional reason.

Reply:
Dropped and moved to v0.7